Main.java

```diff
@@ -1,7 +1,8 @@
 public class Main {
     public static void main(String[] args) {
-        User us = new User("firstname","lastname","email","password");
-        us.saying();
-
+        User us = new User("firstname", "lastname", "email", "password");
```

Comment:
use real values, like your name, your last name, your email, and some random password